add-content.md

```diff
@@ -10,19 +10,20 @@ Prior to adding or editing content, all additions and changes must be approved a
 1. Create a new RAC GitHub repository for the documentation if one does not exist using the [docs-template](https://github.com/RockefellerArchiveCenter/docs-template) repository template. Contact an RAC employee with administrative access to the RAC GitHub to enable this action. See [Create a Repo](/docs-guide/using-github#create-a-repo).
 2. Create a branch called `development` in the target repository.
 3. Commit new content or changes to the `development` branch using short, descriptive commit messages that describe the changes like "add config values," "fix typo," or "change template text". Use this descriptive commit message instead of the default "update [file]" message provided in GitHub.
-4. Review the changes on the [RAC development site](docs.dev.rockarch.org).
+4. Review the changes on the [RAC development site](https://docs-internal.dev.rockarch.org/).
 5. Submit a pull request to merge `development` to the `base` branch of the GitHub repo.
 6. As per the [Content Approval Policy](https://docs.rockarch.org/docs-policy/), each pull request will be reviewed by the relevant Assistant Director and the Director of Archives, as necessary. If a determination is made to either convert documentation from public to private, or to remove it completely, the Assistant Director for Digital Strategies or any other RAC employee with administrative access to the RAC GitHub account should be notified to enable this action.
-7. Once a pull request is reviewed and approved, merge the branch into the `base` branch where it will be incorporated into docs.rockarch.org.
+7. Once a pull request is reviewed and approved, merge the branch into the `base` branch where it will be incorporated into the site.
 
 ## Required Files
```

Comment:
The checkboxes below look kind of weird when combined with the bullet list. Should we just remove them?